mnefun/_mnefun.py

```diff
@@ -2330,3 +2330,117 @@ def plot_reconstruction(evoked, origin=(0., 0., 0.04)):
     axs[2, 0].set_ylabel('Reconstruction')
     fig.tight_layout()
     return fig
+
+
+def chpi_snr_raw(raw, winlen, nharm, stop=None):
+    """ Compute and plot cHPI SNR from raw data.
+
+    Parameters
+    ----------
+
+    winlen : int
+        Length of window to use. Longer windows will naturally include more
+        low frequency power, resulting in lower SNR.
+
+    nharm : int
+        Number of line frequency harmonics to include in the model.
+
+    stop : int
+        If specified, stop before the end of file (seconds)
```

Comment:
See how other docstrings are formatted in MNE-Python and try to follow it as closely as possible. It should probably also return the Matplotilb figure (see other MNE-Python viz functions)